tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -253,10 +253,10 @@ version of the raw dataset will probably go away.
     <!-- Lake data from 2018 to 2023 is copied from 2017 -->
     <!-- Urban data from 2016 to 2023 is copied from 2015 -->
```

Comment:
Should remove line 254 soon, since urban data is not longer copied from 2015 in this version.

Reply:
This is done. Creating surface data now.

Reply:
Yes, this is OK for this first go through.

We really need to remove the Deracho specific paths here, but it looks like that was there beforehand. This is something we'll want to fix in a future version. And we'll want to move these directories over to the inputdata location.

So good to go for the first pass!

Reply:
agreed, I thought after we evaluate these results we can move raw data to input data and then create the full suite of surface data in one clean go.  This draft PR was mainly to facilitate getting eyes on the xml file quickly so I could create the datasets.